<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGame</class>
 <widget class="QMainWindow" name="MainGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>410</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>65536</width>
    <height>65536</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Mine Sweeper</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/resource.qrc">
    <normaloff>:/Images/icon.ico</normaloff>:/Images/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>344</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>开始(S)</string>
    </property>
    <addaction name="actionEasy"/>
    <addaction name="actionNormal"/>
    <addaction name="actionHigh"/>
    <addaction name="actionCustom"/>
    <addaction name="separator"/>
    <addaction name="actionPause"/>
    <addaction name="actionRestart"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="actionRecord"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionInstruction"/>
    <addaction name="actionFeedback"/>
   </widget>
   <addaction name="menuStart"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionEasy">
   <property name="text">
    <string>初级(E)</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="text">
    <string>中级(N)</string>
   </property>
  </action>
  <action name="actionHigh">
   <property name="text">
    <string>高级(H)</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>重新开始(R)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于(A)</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="text">
    <string>自定义(C)</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="text">
    <string>排行榜(R)</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>暂停(P)</string>
   </property>
  </action>
  <action name="actionInstruction">
   <property name="text">
    <string>说明(I)</string>
   </property>
  </action>
  <action name="actionFeedback">
   <property name="text">
    <string>反馈(F)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
